src/js/worklets/bottom_sheet.js

```diff
@@ -6,10 +6,10 @@ export function useTranslateY(initialTranslationY, bottomSheetDy, panY) {
   })
 }
 
-export function useBackgroundOpacity(translateY, backgroundHeight, windowHeight) {
+export function useBackgroundOpacity(translateY, backgroundHeight, windowHeight, opacity) {
   return useDerivedValue(() => {
-    const opacity = ((translateY.value - windowHeight) / -backgroundHeight) * 0.5
+    const calculated_opacity = ((translateY.value - windowHeight) / -backgroundHeight) * opacity
```

Comment:
Is preferable to use camelCase for variable names.

https://www.w3schools.com/js/js_conventions.asp

Reply:
good point.